### ISSUE TITLE
HTML validation error: Logo <img> tag must have alt attribute

### ISSUE THREAD

**Issue by justinsteven** [rakuishi/hugo-zen#8]
The logo <img> tag should have an alt attribute to pass HTML validation checks. I didn't submit a PR as I don't know what you'd prefer to have as the alt text.

https://github.com/rakuishi/hugo-zen/blob/e116649e0c2da17e15bd436f798b8ceb6ac2d0c0/layouts/partials/header.html#L21

**Comment by rakuishi** [comment]
I added alt attribute! :-D
